vault_program/src/initialize_vault.rs

```diff
@@ -124,6 +157,10 @@ pub fn process_initialize_vault(
             vault_bump,
             slot,
         )?;
+
+        // Mint initial VRT supply
```

Comment:
the word "Mint" is a bit misleading here cause it's not actually creating any VRT

Reply:
whats the reason for not actually minting any where? i'm worried this might be a footgun in the future - if we start looking at the vrt_mint supply in the future then it'll be off by INITIALIZATION_TOKEN_AMOUNT

Reply:
That's a good point, can mint to some vault PDA such that the tokens are never usable. This would also fix the issues in the tests where I have to adjust by INITIALIZATION_TOKEN_AMOUNT

Reply:
should we call mint_with_fee here?

Reply:
https://solscan.io/account/11111111111111111111111111111111

Reply:
use mint_to - and setup such that vault.mint_with_fee has no fees

Reply:
Done